package.json

```diff
@@ -61,11 +63,12 @@
     "flux": "2.1.1",
     "fuse.js": "2.6.1",
     "get-video-id": "2.1.0",
+    "globby": "3.0.1",
     "gridicons": "1.0.0",
     "hard-source-webpack-plugin": "0.0.42",
     "he": "0.5.0",
     "html-loader": "0.4.0",
-    "i18n-calypso": "1.7.0",
+    "i18n-calypso": "Automattic/i18n-calypso#add/cli-globs",
```

Comment:
Waiting for https://github.com/Automattic/i18n-calypso/pull/36 to be merged and released.